src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -507,7 +511,7 @@ export const WorkflowView = _.flow(
                   } = modifiedConfig;
                   // will only match if the current root entity type comes from a snapshot
                   const snapshot = _.find({ metadata: { name: modifiedConfig.dataReferenceName } }, availableSnapshots);
-                  Ajax().Metrics.captureEvent(Events.workflowLaunch, {
+                  void Metrics().captureEvent(Events.workflowLaunch, {
```

Comment:
Does the void make it clear that we don't care about awaiting?

Reply:
correct, and it removes a lint warning for the same reason.

Reply:
Cool! It would be good to do it consistently then.

Reply:
ok yeah, I see a few I missed in other PR files.... I'll fix um now